### DISCUSSION TITLE
Kotlin Language not showing in Language Section

### DISCUSSION THREAD

**Discussion by dev-aniketj** [lowlighter/metrics#1366] [Q&A]
Kotlin Showing in **Recently Used Language** section but not showing in **Mostly Used Language** section,
fix this issue

**Comment by jayvdb** [comment]
Try using https://github.com/lowlighter/metrics/tree/master/source/plugins/lines to show which repos are the "top" repos, which may indicate repos which are skewing your top language results.  You can see that plugin working on my profile, generated from my [`lines.svg`](https://github.com/jayvdb/jayvdb/blob/main/lines.svg)
